docs/mindsdb-docs/docs/sql/api/select.md

```diff
@@ -2,47 +2,93 @@
 
 ## Description
 
-The `#!sql SELECT` statement is used to get a predictions from the model table. The data is not persistent and is returned on the fly as a result-set. The basic syntax for selecting from the model is:
+The `#!sql SELECT` statement fetches predictions from the model table. The data is returned on the fly and not saved.
+
+But there are ways to save predictions data! You can save your predictions as a view using the [`CREATE VIEW`](/sql/create/view/) statement. Another way is to insert your predictions into a table using the [`INSERT INTO`](/sql/api/insert/) statement.
 
 ## Syntax
 
+### Single Prediction
+
+Here is the syntax for fetching a single prediction from the model table:
+
 ```sql
 SELECT [target_variable], [target_variable]_explain
 FROM mindsdb.[predictor_name]
 WHERE [column]=[value] 
 AND [column]=[value];
 ```
 
-!!! warning "Grammar matters"
-    Ensure that there are no spaces between the column name, equal sign and value. Ensure to not use any quotations for numerical values and singular quotes for strings.
+!!! warning "Grammar Matters"
+    Here are some points to keep in mind while writing queries in MindsDB:<br/>
+    &nbsp;&nbsp;&nbsp;1. Ensure that there are no spaces between `[column]` and `=` and between `=` and `[value]`.<br/>
```

Comment:
That is obsolete. For me queries is working with any count of spaces anywhere.

Reply:
Instead of p1. we can add:
`[column]=[value]` may be joined only via `AND`

Reply:
I will change point 1 to the following:
> The `[column]=[value]` pairs may be joined by `AND` or `OR` keywords.